src/ui/styles/styles.scss

```diff
@@ -1,4 +1,4 @@
-@import 'node_modules/govuk-frontend/dist/govuk/all';
+@import '../node_modules/govuk-frontend/dist/govuk/all';
```

Comment:
This doesn't look right, have you run `npm run build` in the UI with these changes?

Reply:
yes build fails without the ../

Reply:
on the newest version of sass-loader